cpp/include/cugraph_c/sampling_algorithms.h

```diff
@@ -36,8 +36,78 @@ typedef struct {
   int32_t align_;
 } cugraph_random_walk_result_t;
 
+/**
+ * @brief  Compute uniform random walks
+ *
+ * @param [in]  handle       Handle for accessing resources
+ * @param [in]  graph        Pointer to graph.  NOTE: Graph might be modified if the storage
+ *                           needs to be transposed
+ * @param [in]  sources      Array of source vertices
+ * @param [in]  max_length   Maximum length of the generated path
+ * @param [in]  result       Output from the node2vec call
+ * @param [out] error        Pointer to an error object storing details of any error.  Will
+ *                           be populated if error code is not CUGRAPH_SUCCESS
+ * @return error code
+ */
+cugraph_error_code_t cugraph_uniform_random_walks(
+  const cugraph_resource_handle_t* handle,
+  cugraph_graph_t* graph,
+  const cugraph_type_erased_device_array_view_t* sources,
```

Comment:
`sources` == `start_vertices` in the code above? If yes, better use the same variable name?

Reply:
Same for the functions below.

Reply:
Just pushed an update for this.  Also updated in implementation and test files.